sensu/plugins/check-network-traffic.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
```

Comment:
This was copied from one of the other checks in the repo, but I may have just chosen an older one.